scripts/s3_bucket_setup.R

```diff
@@ -12,3 +12,8 @@ put_bucket(new_bucket_name, region='us-west-2', acl='private') # gives error if
 
 # this command posted the data (took 1.5 hrs)
 put_object(file='data/nhgis0002_shape.zip', object='IPUMS_NHGIS_counties.zip', bucket='viz-water-use-15')
+
+# this command posts the USGS WU mock up data from Google Drive (see pinned items in water-use-fy18-viz Slack channel)
+# had to delete the first row of info before uploading (version num + authors)
```

Comment:
interesting! do you think that was a constraint on the AWS side or on the aws.s3 side?

Reply:
lindsay clarifies: " it uploaded just fine without doing that. I just wanted to easily use the built in excel reader within vizlab and not have to add a whole extra reader just to skip line 1"